### PR TITLE
refactor: simplify utils and how we check for differences

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: packages/svelte-vega/src/VegaEmbed.svelte [hunk 1]
```diff
@@ -99,8 +91,12 @@
   });
 
   async function createView() {
+    clearView();
```
Comment (domoritz) [inline packages/svelte-vega/src/VegaEmbed.svelte:94]: I added this to make sure we always clear before we create a new view.

File: packages/svelte-vega/src/VegaEmbed.svelte [hunk 2]
```diff
@@ -99,8 +91,12 @@
...
+
     const finalSpec = combineSpecWithDimension(spec, options);
+
     try {
+      console.info("Creating new view");
```
Comment (domoritz) [inline packages/svelte-vega/src/VegaEmbed.svelte:99]: This helps with debugging.

File: packages/svelte-vega/src/utils.ts [hunk 3]
```diff
@@ -3,6 +3,8 @@ import { vega } from "vega-embed";
 import equal from "fast-deep-equal";
 import type { VisualizationSpec, EmbedOptions } from "vega-embed";
 
+export const WIDTH_HEIGHT = new Set(["width", "height"]);
+
```
Comment (Sparkier) [inline packages/svelte-vega/src/utils.ts:7]: I moved this to constants. I think this fits better.

**Review by Sparkier — APPROVED** [review]

**Pull request merged**